launchpad-to-stargate/tutorial/01-introduction.md

```diff
@@ -16,19 +16,19 @@ Install it by entering:
 curl https://get.starport.network/starport@v0.13.1! | bash
 ```
 
-We will be migrating the Proof of File existence tutorial from Launchpad to Stargate. At the time of writing, Launchpad is at `v0.39.1` and Stargate is `v0.40.0` of the Cosmos SDK.
+We will be migrating the Proof of File existence tutorial from v0.39 to v0.40+ Stargate. At the time of writing, the version you migrate from is `v0.39.1` and Stargate is `v0.40.0` of the Cosmos SDK.
 
 In order to obtain the code for the Proof of File Existence application, you can follow the tutorial [here](https://tutorials.cosmos.network/proof-of-file-existence/tutorial/01-intro.html), or clone [this](https://github.com/cosmos/sdk-tutorials/tree/master/proof-of-file-existence/pofe) repository.
```

Comment:
```suggestion
To get the code for the Proof of File Existence (PoFE) application, clone the [pofe repo](https://github.com/cosmos/sdk-tutorials/tree/master/proof-of-file-existence/pofe).
```

Reply:
Tip: provide only one method to accomplish a task. I selected the repo clone method, since the linked topic didn't provide much useful information